README.md

```diff
@@ -440,13 +471,34 @@ Examples:
 ## Deleting a database
 
 ```js
-  db.delete(dbName).then(function (e) {
+  db.delete(dbName).then(function (ev) {
       // Should have been a successful database deletion
   }, function (err) {
       // Error during database deletion
   });
 ```
 
+As with the `open` operation, a `delete` operation will not be able to
+execute so long as there are already opened blocking connections
+(i.e., those allowing for database modification) which are open elsewhere
+in the browser. You can recover as follows:
+
+```js
+  db.delete(dbName).catch(function (err) {
```

Comment:
Since we're using the gated code blocks there doesn't need to be the 4 spaces of indentation first. Can you collapse it back